core/src/test/scala/unit/kafka/security/authorizer/AclEntryTest.scala

```diff
@@ -36,13 +37,34 @@ class AclEntryTest {
 
   @Test
   def testAclJsonConversion(): Unit = {
+    val objectMapper = new ObjectMapper()
+    val jsonNode = objectMapper.readTree(AclJson)
+
     val acl1 = new AclEntry(new AccessControlEntry(new KafkaPrincipal(KafkaPrincipal.USER_TYPE, "alice").toString, "host1", READ, DENY))
     val acl2 = new AclEntry(new AccessControlEntry(new KafkaPrincipal(KafkaPrincipal.USER_TYPE, "bob").toString, "*", READ, ALLOW))
     val acl3 = new AclEntry(new AccessControlEntry(new KafkaPrincipal(KafkaPrincipal.USER_TYPE, "bob").toString, "host1", READ, DENY))
+    val expectedAcls = new util.HashSet[AclEntry](util.Arrays.asList(acl1, acl2, acl3))
+
+    val aclsNode = jsonNode.get("acls")
+    val acls = new util.HashSet[AclEntry]()
+
+    aclsNode.forEach(aclNode => {
```

Comment:
Why did we add this extra code?

Reply:
To maintain consistency with the existing test structure which uses `Json.encodeAsBytes(AclEntry.toJsonCompatibleMap(acls))` for converting between string and AclEntry sets, we should modify our test to use ObjectMapper while achieving the same validation goals.

Reply:
`toJsonCompatibleMap` was completely removed, so I don't see what we're trying to test here.

Reply:
make sense to me, I think we should remove this test, becsue `fromBytes` only use in this test, there are not any production code use it.